projects/packages/my-jetpack/_inc/components/product-card/index.jsx

```diff
@@ -255,18 +242,8 @@ const ProductCard = props => {
 		onDeactivateStandalone();
 	}, [ slug, onDeactivateStandalone, recordEvent ] );
 
-	const CardWrapper = isAbsent
```

Comment:
Forcing the card to be a `div` since having an interactive element (`a` or `button`) inside another is neither valid HTML nor accessible.